### PR TITLE
Update to 0.9.0

### PR DESCRIPTION
**Pull request by blasten**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -1,8 +1,34 @@
...
+    "polymer": "polymer/polymer#v0.9.0-rc.1",
+    "iron-range-behavior": "PolymerElements/iron-range-behavior#0.9.0"
+  },
+  "devDependencies": {
+    "iron-component-page": "PolymerElements/iron-component-page#^0.9.0",
+    "test-fixture": "PolymerElements/test-fixture#^0.9.0",
+    "webcomponentsjs": "webcomponents/webcomponentsjs#^0.6.0",
+    "paper-button": "PolymerElements/paper-button#^0.9.0"
+  },
+  "resolutions": {
```
Comment (cdata) [inline bower.json]: This should not be necessary

Comment (blasten) [inline bower.json]: the resolutions you mean?

Comment (cdata) [inline bower.json]: Yeah

Comment (blasten) [inline bower.json]: ok, fixing now.  should I merge it?

File: paper-progress.html [hunk 2]
```diff
@@ -27,98 +28,180 @@
...
+<script>
+  Polymer({
+
+    is: 'paper-progress',
+
+    behaviors: [
+      Polymer.IronRangeBehavior
+    ],
+
+    enableCustomStyleProperties: true,
```
Comment (cdata) [inline paper-progress.html]: This is no longer needed in 0.9

Comment (notwaldorf) [inline paper-progress.html]: Not needed anymore

File: test/basic.html [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+
+
+  <test-fixture id="trivialProgress">
+    <template>
+      <paper-progress></paper-progress>
+    </template>
+  </test-fixture>
+
+  <script>
+    suite('<paper-progress>', function() {
```
Comment (cdata) [inline test/basic.html:34]: Did you run the tests locally, by any chance?

File: paper-progress.html (hunk 2 above)
Comment (blasten) [inline paper-progress.html]: I forgot about it. awesome

File: test/basic.html [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+          // test clamp value
+          range.value = 60.1;
+          asyncPlatformFlush(function() {
+            assert.equal(range.value, 60);
+            done();
+          });
+        });
+      });
+
+
```
Comment (notwaldorf) [inline test/basic.html]: nit: too many new lines :)

File: bower.json (hunk 1 above)
Comment (cdata) [inline bower.json]: For pull requests, you should wait until someone says "LGTM :+1:" or something like that before you merge.

File: paper-progress.html [hunk 5]
```diff
@@ -27,98 +28,180 @@
...
-<link rel="import" href="../core-range/core-range.html">
+<style is="custom-style">
+  * {
+    --paper-progress-active-color: #0f9d58;
+    --paper-progress-secondary-color: #f8bbd0;
+    --paper-progress-container-color: #c8c8c8;
+  }
+</style>
+
+<link rel="import" href="../iron-range-behavior/iron-range-behavior.html">
```
Comment (notwaldorf) [inline paper-progress.html]: nit: can you move the imports before the style, please?

**Comment by notwaldorf** [comment]
Also, could you please squash all your commits in one?

File: bower.json [hunk 6]
```diff
@@ -1,8 +1,34 @@
...
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/PolymerElements/paper-progress.git"
+  },
   "dependencies": {
-    "core-range": "Polymer/core-range#master",
-    "paper-button": "Polymer/paper-button#master"
+    "polymer": "polymer/polymer#v0.9.0-rc.1",
+    "iron-range-behavior": "PolymerElements/iron-range-behavior#0.9.0"
```
Comment (cdata) [inline bower.json]: Has this been updated to 0.9 yet?

File: test/basic.html (hunk 3 above)
Comment (cdata) [inline test/basic.html:34]: Resolved f2f.

File: bower.json (hunk 6 above)
Comment (blasten) [inline bower.json]: @cdata Yes

**Comment by blasten** [comment]
Could we merge this? @cdata @notwaldorf

File: demo/index.html [hunk 7]
```diff
@@ -1,84 +1,84 @@
...
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE
+The complete set of authors may be found at http://polymer.github.io/AUTHORS
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS
 -->
 <!doctype html>
 <html>
 <head>
   <title>paper-progress</title>
```
Comment (notwaldorf) [inline demo/index.html]: nit: paper-progress demo

File: demo/index.html [hunk 8]
```diff
@@ -1,84 +1,84 @@
...
-  
+
+  <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
+
+  <link rel="import" href="../paper-progress.html">
+  <link rel="import" href="../../paper-button/paper-button.html">
+
+  <style is="custom-style">
+
     body {
```
Comment (notwaldorf) [inline demo/index.html]: You can use `demo.css` from `paper-styles`, and you can nuke this body selector

File: demo/index.html [hunk 9]
```diff
@@ -1,84 +1,84 @@
...
       display: block;
       width: 100%;
       padding: 25px 0;
     }
-    
-    paper-progress.pink::shadow #activeProgress {
-      background-color: #e91e63;
+
+    paper-progress.pink {
+      --paper-progress-active-color: #e91e63;
```
Comment (notwaldorf) [inline demo/index.html]: Can you use a custom colour from `default-theme.html`? Like `--accent-color` (and for the other ones)?

Comment (blasten) [inline demo/index.html]: Is `--paper-progress-active-color: var(--accent-color);`supported? would that be all?

Comment (notwaldorf) [inline demo/index.html]: I've used it in the `paper-checkbox` demo, so it should work. Does it not work for you?

Comment (blasten) [inline demo/index.html]: Yeah. I wasn't sure.  I ended up using`--google-*` colors as those are closer to the values that it had. Can you check again please? thanks

File: bower.json [hunk 10]
```diff
@@ -1,8 +1,29 @@
...
+  },
   "dependencies": {
-    "core-range": "Polymer/core-range#master",
-    "paper-button": "Polymer/paper-button#master"
+    "polymer": "polymer/polymer#v0.9.0-rc.1",
+    "iron-range-behavior": "PolymerElements/iron-range-behavior#0.9.0",
+    "paper-styles": "PolymerElements/paper-styles#^0.9.0"
+  },
+  "devDependencies": {
+    "iron-doc-viewer": "PolymerElements/iron-doc-viewer#^0.9.0",
```
Comment (cdata) [inline bower.json]: `iron-component-page`

File: test/index.html [hunk 11]
```diff
@@ -0,0 +1,25 @@
...
+  <script src="../../web-component-tester/browser.js"></script>
+</head>
+<body>
+  <script>
+    WCT.loadSuites([
+      'basic.html'
+    ]);
+  </script>
+</body>
+</html>
```
Comment (cdata) [inline test/index.html]: Newline @ EOF?

File: paper-progress.html [hunk 12]
```diff
@@ -27,98 +28,179 @@
...
-<link rel="import" href="../core-range/core-range.html">
+<link rel="import" href="../paper-styles/paper-styles.html">
+<link rel="import" href="../iron-range-behavior/iron-range-behavior.html">
+<link rel="import" href="../iron-flex-layout/classes/iron-flex-layout.html">
+
+<style is="custom-style">
+  * {
+    --paper-progress-active-color: var(--google-green-500);
+    --paper-progress-secondary-color: var(--google-red-100);
+    --paper-progress-container-color: var(--google-grey-300);
```
Comment (cdata) [inline paper-progress.html]: I wonder if these should be integrated into https://github.com/PolymerElements/paper-styles/blob/master/default-theme.html somehow..

File: test/index.html (hunk 11 above)
Comment (blasten) [inline test/index.html]: added

File: bower.json (hunk 10 above)
Comment (blasten) [inline bower.json]: changed

File: paper-progress.html (hunk 12 above)
Comment (blasten) [inline paper-progress.html]: let me know. :+1:

File: paper-progress.html [hunk 13]
```diff
@@ -27,98 +28,171 @@
...
-      background-color: #64ffda;
+    paper-progress {
+      --paper-progress-container-color: #64ffda;
     }
 
 @group Paper Elements
 @element paper-progress
-@extends core-range
 @homepage github.io
 -->
```
Comment (cdata) [inline paper-progress.html:54]: Sorry for the late catch, but can you move the comments below the `import`s? If the comment does not appear next to the `dom-module`, it won't show up in `iron-doc-viewer` /  `iron-component-page`.

Comment (blasten) [inline paper-progress.html:54]: moved!

File: demo/index.html (hunk 7 above)
Comment (blasten) [inline demo/index.html]: fixed

File: demo/index.html (hunk 8 above)
Comment (blasten) [inline demo/index.html]: Added!

File: bower.json [hunk 14]
```diff
@@ -1,8 +1,29 @@
...
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/PolymerElements/paper-progress.git"
+  },
   "dependencies": {
-    "core-range": "Polymer/core-range#master",
-    "paper-button": "Polymer/paper-button#master"
+    "polymer": "polymer/polymer#v0.9.0-rc.1",
+    "iron-range-behavior": "PolymerElements/iron-range-behavior#0.9.0",
```
Comment (cdata) [inline bower.json]: I think this needs to be pointing to `#^0.9.0` (you are missing the caret).

**Comment by cdata** [comment]
There is one more detail to address in `bower.json`. After that, it LGTM :+1:

File: bower.json (hunk 14 above)
Comment (blasten) [inline bower.json]: :+1:

**Pull request merged**